tests/storage-volumes-vm

```diff
@@ -272,6 +288,16 @@ do
 
 		lxc storage volume unset "${poolName}" virtual-machine/v2 security.shared
 
+		# Snapshots can be attached to profiles without security.shared because they
+		# will be mounted read-only
+		lxc profile create v2-snap0
+		lxc storage volume attach-profile "${poolName}" virtual-machine/v2/snap0 v2-snap0 v2-rs
+
+		lxc profile add v1 v2-snap0
```

Comment:
Is it worth checking if the volume is hotplugged into `v1`?

Reply:
Yes, I've found ~2 bugs since doing so.